Library/Formula/git-annex.rb

```diff
@@ -23,6 +23,7 @@ class GitAnnex < Formula
   depends_on "libidn"
   depends_on "gnutls"
   depends_on "gmp"
+  depends_on "wget"
```

Comment:
It is a runtime dependency used to transfer files from different so called special remotes. From reading the documentation it seems that either curl or wget should be needed, but that wget recommended. So ideally the dependency should be changed to :recommended. 

However, the latest version does not seem to build without an installed wget. I just reported this as a [bug](http://git-annex.branchable.com/bugs/Build_fails_when_no_wget_avalible/) upstreams.